tests/acm_observability/README.md

```diff
@@ -0,0 +1,16 @@
+# ACM Observability tests
+
+To run ACM Observability tests make sure that the `KUBEADMIN_TOKEN` environment variable is set or provide
+it within the execution command along with the target cluster name.
+
+To obtain the kubeadmin token via Openshift cluster console (admin privileges needed):
+
+```text
```

Comment:
The instruction should be how to get it via CLI, not UI

Reply:
done